Converters/Autocad/Speckle.Converters.AutocadShared/ToSpeckle/Geometry/HatchToSpeckleConverter.cs

```diff
@@ -112,8 +105,93 @@ private ADB.Curve PolylineFromLoop(ADB.HatchLoop loop)
         }
         return polyline;
       }
+
+      // if .Polyline is null, read from .Curves
+      if (loop.Curves.Count == 0)
+      {
+        throw new ConversionException($"Hatch loop doesn't contain any segments.");
+      }
+
+      if (loop.Curves.Count > 1) // handle the multi-segment case only with Line segments (not able to produce a Hatch Loop with multiple segments of other types)
+      {
+        foreach (var lineSegment in loop.Curves)
+        {
+          // for each segment, skip the last point: it will be added as a start Point of the next segment. Otherwise, they will overlap and make the curve invalid
+          count = lineSegment is AG.LineSegment2d line
+            ? TryAddPointToPolyline(vertices, polyline, line.StartPoint, 0, count)
```

Comment:
Is it necessary to convert linesegment2d into polyline? Does it not work when using Line directly?